README.md

```diff
@@ -7,6 +7,29 @@
 Horovod is a distributed training framework for TensorFlow. The goal of Horovod is to make distributed Deep Learning
 fast and easy to use.
 
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Why not traditional Distributed TensorFlow?](#why-not-traditional-distributed-tensorflow)
+- [Install](#install)
+- [Concepts](#concepts)
+- [Usage](#usage)
+- [Running Horovod](#running-horovod)
+- [Keras](#keras)
+- [Estimator API](#estimator-api)
+- [mpi4py](#mpi4py)
+- [Inference](#inference)
+- [Tensor Fusion](#tensor-fusion)
+- [Analyzing Horovod Performance](#analyzing-horovod-performance)
+- [Guides](#guides)
+- [Troubleshooting](#troubleshooting)
+  - [Citation](#citation)
```

Comment:
Let's change Citation, Publications, References to first level headings (##), so they stop being subsections of Troubleshooting.

Reply:
Good catch :)